www/dusv03/Band.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang=cs>
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="description" content="Neoficiální fanouškovská stránka hudební skupiny Powerwolf" />
+    <meta name="author" content="Vladimír Dušek" />
+    <meta name="keywords" content="Powerwolf, metal, skupina, powermetal" />
+    <link rel="icon" href="images/Powerwolf_Logo.ico">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Powerwolf</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="styles/SPstyles.css" />
+    <link rel=stylesheet href="styles/print.css" media="print" />
+</head>
+
+<body>
+    <header>
+        <div class="header1">Powerwolf</div>
+        <nav>
+            <ul class="list_nav">
+                <li class="paige_nav"><a href="Mainpaige.html">Hlavní strana</a></li>
+                <li class="paige_nav"><a href="Tour.html">Koncerty</a></li>
+                <li class="paige_nav"><a href="Band.html">Kapela</a></li>
+                <li class="paige_nav"><a href="Discography.html">Diskografie</a></li>
+                <li class="paige_nav"><a href="Photos.html">Foto</a></li>
+                <li class="paige_nav"><a href="About.html">O stránce</a></li>
+                <li class="paige_nav"><a href="https://www.powerwolf.net/">Oficiální stránka</a></li>
+            </ul>
+            <div class="clear"></div>
```

Comment:
proc je tady clear?

Reply:
protože když tam není a dojde ke snížení šířky stránky dostane se navigace na jeden řádek k nadpisu, clear asi není nutný ale nějaký oddělovač je potřeba ale takhle to žádné problémy nezpůsobuje